hedyorg/hedy
# 🪲 FIx not-in command missing from level 8 and up

- Fixes #3934
- Add support for the not-in-list command starting from level 8
- Add syntax highlighting for the not-in-list command starting from level 5

**How to test**

Follow these steps to verify this PR works as intended:

The language change is covered with unit tests. Ensure that the tests check all cases including static type checking.
* Run Hedy locally and execute the following snippets for the specified levels. In all cases, the output of the program should be 'not in' and the syntax 'not in' should be highlighted in pink in the editor:
```
# level 5
colors is green, yellow
choice is red
if choice not in colors print 'not in'
``` 

```
# level 8
colors is green, yellow
choice is red
if choice not in colors
  print 'not in'
```

```
# level 12
colors is 'green', 'yellow'
choice is 'red'
if choice not in colors
  print 'not in'
```

```
# level 16
colors is ['green', 'yellow']
choice is 'red'
if choice not in colors
  print 'not in'
```
